### PR TITLE
Feature/sagemaker post

### PR DESCRIPTION
**Pull request by mkianpour**
Post about AWS SageMaker. I broke it into 2 parts

### CODE REVIEW

File: _posts/2019-02-19-SageMaker-In-Action.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+      "__label__technical"
+    ]
+  }
+]
+```
+I tried some more tweets and overall I was happy with the results.
+We can continue more test but now we went over the whole cycle of SageMaker which was the purpose of this blog post. If we are not satisfied with the results, we can try to achieve better results by reviewing and refining training dataset or changing hyperparameters for the algorithm or even better algorithms but again these are data science topics and out of the scope of this post.
+
+## Delete the Endpoint
+Remember that ML instances are expensive. So, if you are done with the experiment and it's not in production! better to delete the endpoint before getting surprised after seeing the invoice!
```
Comment (tvb) [inline _posts/2019-02-19-SageMaker-In-Action.md]: why the exclamation mark after `production`?

File: _posts/2019-02-19-SageMaker-In-Action.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+data_location = 's3://{}/{}'.format(bucket, data_key)
+
+tweets = pd.read_csv(data_location)
+
+tweets.to_csv('tweets-labeled.csv', sep=',', header=None, index=False)
+
+prefix = 'twitter-data/marketing-model'
+
+index_to_label = {}
+#with open("dbpedia_csv/classes.txt") as f:
```
Comment (tvb) [inline _posts/2019-02-19-SageMaker-In-Action.md]: remove unused comments

File: _posts/2019-02-19-SageMaker-In-Action.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+import json
+import boto3
+import pandas as pd
+
+sess = sagemaker.Session()
+
+role = get_execution_role()
+print(role) # This is the role that SageMaker would use to leverage AWS resources (S3, CloudWatch) on your behalf
+
+bucket = "XYZ" # Replace with your own bucket name if needed
```
Comment (tvb) [inline _posts/2019-02-19-SageMaker-In-Action.md]: Use an ENV var instead?

Comment (mkianpour) [reply _posts/2019-02-19-SageMaker-In-Action.md]: it's proof of concept. you can create Notebooks with CloudFormation but here not used and ENV var is not meaningful.

**Comment by mkianpour** [comment]
@kbessas I couldn't merge. Not sure how to proceed.

**Pull request merged**